src/HotChocolate/Core/src/Types.Scalars/LatitudeType.cs

```diff
@@ -0,0 +1,219 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.RegularExpressions;
+using HotChocolate.Language;
+
+namespace HotChocolate.Types
+{
+    /// <summary>
+    /// The `LatitudeType` scalar represents a valid decimal degrees latitude number.
+    /// <a href="https://en.wikipedia.org/wiki/Latitude">Read More</a>
+    /// </summary>
+    public class LatitudeType : ScalarType<double, StringValueNode>
+    {
+        /// <summary>
+        /// Initializes a new instance of <see cref="LatitudeType"/>
+        /// </summary>
+        public LatitudeType()
+            : this(
+                WellKnownScalarTypes.Latitude,
+                ScalarResources.LatitudeType_Description)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="LatitudeType"/>
+        /// </summary>
+        public LatitudeType(
+            NameString name,
+            string? description = null,
+            BindingBehavior bind = BindingBehavior.Explicit)
+            : base(name, bind)
+        {
+            Description = description;
+        }
+
+        protected override bool IsInstanceOfType(StringValueNode valueSyntax)
+        {
+            if(Latitude.TryDeserialize(valueSyntax.Value, out var value))
+            {
+                return value is > Latitude._min and < Latitude._max;
+            }
+
+            return false;
```

Comment:
I think
```suggestion
            return Latitude.TryDeserialize(valueSyntax.Value, out _);
```
 is enough :)

Reply:
> I think
> 
> is enough :)

don't we need to ensure the `deserialized` value is within the range of -90 to 90?